docs/source/benchmarking.rst

```diff
@@ -71,73 +90,84 @@ Aggregate across simulated communities (when dataset index is `not` unique)
-If you have installed ``autometa`` using ``conda`` then ``gdown`` should already be installed.
-If not, you can install it using ``conda install -c conda-forge gdown`` or ``pip install gdown``.
+.. note::
+    
+    First, F1-scores are determined for each reference genome across all recovered genome bins (specifically for clusters containing any of the reference genome's contigs). Then, each reference genome's F1-score is selected corresponding to the respective cluster with the largest set of the respective reference genome's contigs (by length in bp).
 
-Example for the 78Mbp simulated community
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. code:: bash
+    
+    autometa-download-dataset \
+        --community-type simulated \
+        --community-sizes 78Mbp \
+        --file-names binning.tsv.gz\
+        --dir-path simulated
 
-1. Navigate to the 78Mbp community dataset using the `link <https://drive.google.com/drive/u/2/folders/1McxKviIzkPyr8ovj8BG7n_IYk-QfHAgG>`_ mentioned above.
-2. Get the file ID by navigating to any of the files and right clicking, then selecting the ``get link`` option. This will have a ``copy link`` button that you should use. The link for the metagenome assembly (ie. ``metagenome.fna.gz``) should look like this : ``https://drive.google.com/file/d/15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y/view?usp=sharing``
-3. The file ID is within the ``/`` forward slashes between ``file/d/`` and ``/``, e.g:
+Now running the command to benchmark:
 
 .. code:: bash
 
-    # Pasted from copy link button:
-    https://drive.google.com/file/d/15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y/view?usp=sharing
-    #                 begin file ID ^ ------------------------------^ end file ID
+    autometa-benchmark \
+        --benchmark binning-classification \
+        --predictions 78Mbp/binning.tsv \
+        --reference 78Mbp/reference_assignments.tsv.gz \
+        --output-wide binning_classification_wide.tsv.gz
+
+Specifying multiple results
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-4. Copy the file ID
-5. Now that we have the File ID, you can specify the ID or use the ``drive.google.com`` prefix. Both should work.
+``autometa-benchmark`` provides the flexibility of specifying multiple output files of the same dataset and benchmarking them simultaneously. For example, in case you have done the clustering for a sample using three different binner, you can benchmark all of them simulateously. This is really handy when benchmarking multiple tools on the same dataset.
+
+For clustering
+--------------
 
 .. code:: bash
 
-    file_id="15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y"
-    gdown --id ${file_id} -O metagenome.fna.gz
-    # or
-    gdown https://drive.google.com/uc?id=${file_id} -O metagenome.fna.gz
+    autometa-benchmark \
+       --benchmark clustering \
+       --predictions binner1_78Mbp_output.tsv.gz binner2_78Mbp_output.tsv.gz binner3_78Mbp_output.tsv.gz \
```

Comment:
Why not just list some of the binners outputs from KwanLab/metaBenchmarks?

Reply:
I can, I just didn't want people to upset people. Some people might think that we are specifically targeting or pinpointing the above-mentioned binners.